scenes/menu/preferences_window.gd

```diff
@@ -1,25 +1,53 @@
 extends PopupPanel
 
+signal bgm_check_toggled
+
 @onready var timer_selection = $VBoxContainer/StartingTimersSelection
 @onready var best_of_selection = $VBoxContainer/BestOfSelection
+@onready var enforce_timer_checkbutton = $VBoxContainer/EnforceTimerCheckbutton
+@onready var minimum_time_selection = $VBoxContainer/MinimumTimeSelection
+@onready var bgm_checkbutton = $VBoxContainer/BGMCheckbutton
+@onready var game_sound_checkbutton = $VBoxContainer/GameSoundsCheckbutton
+@onready var ai_first_player_checkbutton = $VBoxContainer/AIFirstPlayerCheckbutton
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
+	GlobalSettings.settings_loaded.connect(display_loaded_settings)
```

Comment:
If we change this later so this scene can be created during the game, we probably won't just have it existing to get this settings_loaded message.
So maybe just call display_loaded_settings() at the end of _ready() to handle the case that this signal isn't going to get emitted after game launch?

Reply:
I had it this way, but there's a timing issue. there's definitely a better way to resolve this, but we can handle creating the pref windows scene later by calling GlobalSettings.load_settings() or whatever it is (where the signal is emitted) after instantiating the preferences window